cif_pow.dic

```diff
@@ -10977,6 +10977,55 @@ save_PD_QPA_CALIB_FACTOR
     _name.object_id               PD_QPA_CALIB_FACTOR
     _category_key.name            '_pd_qpa_calib_factor.phase_id'
 
+    _description_example.case
+;
+         data_phase_1
+             _pd_phase.id   PHASE_A
+             _pd_qpa_calib_factor.I_over_Ic   3.26
+
+         data_phase_2
+             _pd_phase.id   PHASE_B
+             _pd_qpa_calib_factor.I_over_Ic   4.79
+
+         data_phase_3
+             _pd_phase.id   PHASE_C
+             _pd_qpa_calib_factor.I_over_Ic   1.00
+
+         data_diffractogram_block
+             _pd_diffractogram.id   UNKNOWN_DIFFRACTOGRAM
+
+             loop_
+             _pd_phase_mass.phase_id
```

Comment:
`_pd_phase_mass.phase_id` should only have a single value in a data block. In order to show this loop, `_audit.schema` should be `Custom`.